pkgs/applications/emulators/wine/base.nix

```diff
@@ -186,7 +186,7 @@ stdenv.mkDerivation ((lib.optionalAttrs (buildScript != null) {
   passthru = {
     inherit pkgArches;
     tests = { inherit (nixosTests) wine; };
-  };
+  } // (if src ? updateScript then { inherit (src) updateScript; } else {});
```

Comment:
Can we make this unconditional?

Reply:
Oh, we can! I didn't realize staging would simply borrow unstable's `src` and so wouldn't cause an issue here.